history.rst

```diff
@@ -5,6 +5,37 @@
 OMERO version history
 =====================
 
+5.6.7 (March 2023)
+------------------
+
+This release includes the following upgrade of the OMERO.server Java components:
+
+- omero-gateway-java 5.7.2
+- omero-blitz 5.6.2
+- omero-server 5.6.7
+- omero-renderer 5.5.12
+- omero-romio 5.7.2
+- omero-common 5.6.1
+- omero-model 5.6.10
+
+
+Improvements include:
+
+- address performance issues when indexing fileset
+- add option to configure fileset indexing
+- run the PixelDataThread Application events in SYSTEM Thread pool.
+- Declare logback-classic as explicit dependency and set to 1.2.x.
+- an upgrade of Bio-Formats to version 6.12.0
+
+**Note:** This upgrade will invalidate the Bio-Formats Memoizer cache. Please
+see the upgrade guide for further information.
```

Comment:
In addition with the memoizer cache regeneration, the only other point possibly worth discussing is that people might want to consider reindexing their database. 

The choice will be obviously domain/application-specific and is complex to communicate. It would be the case in the case of an instance with filesets with very large number of files e.g. an HCS OMERO where the indexing is likely to be unable to catch up. It could also be relevant in an instance where at least some of the filesets exceed the max size specified by the property and it is important to have consistency in terms of the search results between data indexed prior to and after the upgrade.

Reply:
@sbesson adjustments made